catalog/opensearch/catalog-opensearch-endpoint/src/main/java/org/codice/ddf/opensearch/query/OpenSearchQuery.java

```diff
@@ -334,11 +342,11 @@ public void addTypeFilter(String type, String versions) {
 
     if (versions != null && !versions.isEmpty()) {
       LOGGER.debug("Received versions from client.");
-      String[] typeVersions = versions.split(",");
-      List<Filter> typeVersionPairsFilters = new ArrayList<Filter>();
+      String[] typeVersions = versions.split(OpenSearchParameters.VERSION_DELIMINATOR);
```

Comment:
Might be more performant to compile the regex from `OpenSearchParameters.VERSION_DELIMINATOR` and then call split from the regex instead.